setup.sh

```diff
@@ -21,6 +21,7 @@ sudo apt-get install python-psycopg2 --yes
 sudo apt-get install python-gobject --yes
 sudo apt-get install python-pycurl --yes
 sudo apt-get install postgresql-9.3-postgis-2.1 --yes
+sudo apt-get install landscape-client --yes
```

Comment:
I agree with Nate, but I also think it's outside of the scope of this pull. I'd be hesitant to change anything as this application is unfortunately delicate.